src/Console/BaseCommand.php

```diff
@@ -178,8 +190,12 @@ public function run(array $argv, ConsoleIo $io): ?int
             $io->setInteractive(false);
         }
 
-        /** @var int|null */
-        return $this->execute($args, $io);
+        $this->dispatchEvent('Command.executionStarted', ['args' => $args]);
+        /** @var int|null $result */
+        $result = $this->execute($args, $io);
+        $this->dispatchEvent('Command.executionFinished', ['args' => $args, 'result' => $result]);
```

Comment:
Could we do `Command.beforeExecute` and `Command.afterExecute` to mirror the before/after conventions that we have for controller actions and find operations?

Reply:
My naming skills are - as usual - beneath contempt... This is definitely more consistent with other events in the framework 😁